eq-author/src/enhancers/withPropRenamed.js

```diff
@@ -1,9 +1,10 @@
 import React from "react";
 
-export default (oldName, newName) => Component => props => {
-  const newProps = {
-    [newName]: props[oldName],
-    ...props,
+export default (oldName, newName) => WrappedComponent =>
+  class RemappedComponent extends React.Component {
```

Comment:
You can set the displayName of the component to be something like `withPropRenamed(${WrappedComponent.displayName})` so that it gets that name not this one.